### ISSUE TITLE
Embedded power BI iframes don't do not have accessible names

### ISSUE THREAD

**Issue by jimmy3912msncom** [microsoft/powerbi-client-react#33]
I have a power BI report that I am embedding on our web page. The custom visual and the report itself are iframes. Our accessibility team used [accessibility insights for web](https://accessibilityinsights.io/) version 2.23.0 and detected that we violated the frame-title rule where there is no accessible name in the iframes.

We tried updating to the latest powerbi-client-react version 1.3.1 but the issues persists.

See below for the screenshot including the accessibility details.

The right hand bang error is for the overall iframe and the left hand hang error is for the custom visual in our Power BI report.

![image](https://user-images.githubusercontent.com/6807772/107831441-ed36e900-6d42-11eb-805e-c29de0bfa7c1.png)

**Comment by jimmy3912msncom** [comment]
This seems to be an issue with the Power BI service which they seem to be addressing so closing the issue
